jupyterhub/singleuser/__init__.py

```diff
@@ -28,15 +28,20 @@
         # - extension, if jupyter server 2
         # - older subclass app, otherwise
         try:
-            import jupyter_server
+            import jupyverse_api  # noqa: F401
```

Comment:
The priority here shouldn't change, which would be breaking. Let's add jupyverse as opt-in only via JUPYTERHUB_SINGLEUSER_APP=jupyverse.

Reply:
Thanks, https://github.com/jupyterhub/jupyterhub/pull/4520/commits/f342dd2540fd21abff5c4ae280ee101595cdfa7b implements that, and I also moved all the Jupyverse launching logic to the Jupyverse plugin.
I will add a test to this PR when https://github.com/jupyter-server/jupyverse/pull/335 is merged and released.